documentation/staging/content/managing-domains/accessing-the-domain/status-conditions.md

```diff
@@ -0,0 +1,202 @@
++++
+title = "Status conditions"
+date = 2022-10-24T16:43:45-05:00
+weight = 9
+description = "Monitor domain and cluster resources using operator-generated status conditions."
++++
+
+{{< table_of_contents >}}
+
+### Overview
```

Comment:
I think that we should simplify this overview.  It's enough to say that, "The WebLogic Kubernetes Operator populates status conditions on Domain and Cluster resources to provide high-level status reporting."

If you want to say more than you could note that "Status conditions are a Kubernetes standard mechanism and the conditions generated by the operator are similar to those that Kubernetes provides for Pod and Deployment resources" https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties.

Finally, this might be a good location to talk about `metadata.generation` and `status.observedGeneration` and note that conditions are asynchronously updated as status changes and the generation related fields let the customer know whether the customer know whether the current conditions are about the current contents of the resources or an earlier version.